src/features/labels/Vote.tsx

```diff
@@ -93,15 +95,17 @@ export default function Vote({ item }: VoteProps): React.ReactElement {
           >
             <IonIcon icon={arrowUpSharp} /> {upvotes}
           </Container>
-          <Container
-            vote={myVote}
-            voteRepresented={-1}
-            onClick={async (e) => {
-              await onVote(e, myVote === -1 ? 0 : -1);
-            }}
-          >
-            <IonIcon icon={arrowDownSharp} /> {downvotes}
-          </Container>
+          {downvoteAllowed ? (
+            <Container
+              vote={myVote}
+              voteRepresented={-1}
+              onClick={async (e) => {
+                await onVote(e, myVote === -1 ? 0 : -1);
+              }}
+            >
+              <IonIcon icon={arrowDownSharp} /> {downvotes}
+            </Container>
+          ) : undefined}
```

Comment:
It's different, but forcing total may be a less confusing experience. This implementation would distinguish downvotes coming from other federated sources, which may allow downvotes.

Example `counts` object:
```
{
  "id": 161229,
  "comment_id": 184559,
  "score": -2,
  "upvotes": 2,
  "downvotes": 4,
  "published": "2023-07-20T14:02:40.138416",
  "child_count": 0,
  "hot_rank": 0
}
```

You'd see the 2 upvotes, instead of the -2 total score this way

Reply:
I'll tweak the or when I get back, but I think sticking the down votes back as a number that can't be interacted with might be the correct solution here. Not a "sticking your head in the sand" situation then

Reply:
That sounds like a good solution to me